### PR TITLE
perf-test: Added skottie player

### PR DESCRIPTION
**Pull request by tinyjin**
Integration of Skottie Player from [canvaskit-wasm](https://www.npmjs.com/package/canvaskit-wasm).

[Performance Result]
- Renderer : Skia SW Canvas
- Animation count : 20
- Seed : [link](https://thorvg-perf-test-git-jinny-skottie-thorvg-web.vercel.app/?seed=aXNvbWV0cmljLGZsaXBwaW5nX3BhZ2Usdm9pY2VfcmVjb2duaXRpb24sbWVyZ2luZ19zaGFwZXMsZG93bixjYXRfbG9hZGVyLHRleHQyLGFsaWVuLGxvbG8saW5zdGFfY2FtZXJhLGhvdXJnbGFzcyxlbW9qaV9lbmpveWluZyxjYXRfbG9hZGVyLHNhZF9lbW9qaSxiaXJ0aF9zdG9uZV9sb2dvLHNhZF9lbW9qaSwxMTU1NSxzdGFydHNfdHJhbnNwYXJlbnQsbWF0ZXJpYWxfd2F2ZV9sb2FkaW5nLGFic3RyYWN0X2NpcmNsZQ%3D%3D&player=skia%2Fskottie&count=20) 
- FPS : 25~38FPS
- LightHouse Score : 68

CleanShot 2024-06-13 at 15 03 52@2x

CleanShot 2024-06-13 at 15 44 24@2x

### CODE REVIEW

File: perf-test/app/page.tsx [hunk 1]
```diff
@@ -459,6 +481,16 @@ export default function Home() {
                   ></Player>
                 )
               }
+              {
+                playerId == 4 && (
+                  <SkottiePlayer
+                    lottieURL={anim.lottieURL}
+                    mode={SkottieRenderMode.CPU}
```
Comment (hermet) [inline perf-test/app/page.tsx:488]: Could you briefly compare the perf(memory/fps/lighthouse) between CPU/GPU?  We could choose a better one.

Comment (tinyjin) [reply perf-test/app/page.tsx:488]: The SkottiePlayer component supports GPU canvas. However it only allows 16 animations in my machine, which is maximum hardware concurrency. Still I don't know the reason why, I followed [skia's example code](https://github.com/google/skia/blob/main/tools/skottie-wasm-perf/skottie-wasm-perf.html#L27). Need to figure out to expand the limitation.

So I removed Skia GPU option in this PR.

Comment (hermet) [reply perf-test/app/page.tsx:488]: \+ @kudanai

File: perf-test/components/SkottiePlayer/index.tsx [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+        seek = 0;
+        frame = 0;
+      }
+
+      if (mode === SkottieRenderMode.CPU) {
+        window.requestAnimationFrame(drawFrame);
+        return;
+      }
+
+      setTimeout(() => {
```
Comment (hermet) [inline perf-test/components/SkottiePlayer/index.tsx:92]: could you please remove this unreachable code?

**Review by hermet — APPROVED** [review]

**Pull request merged**